testing/testing_index.adoc

```diff
@@ -0,0 +1,97 @@
+// vim: set syntax=asciidoc:
+[[Testing]]
+== Testing
+:data-uri:
+:toc:
+:toclevels 4:
+:homepage https://github.com/projectatomic/container-best-practices:
+
+=== Purpose of the testing
+
+Container images generally consist of distro packages and some scripts that help to start the container properly. For example in case of MariaDB database, it is set of RPMs with mariadb-server that provides the main functionality, but we need some scripts that handle proper database stack initialization, changes basic configuration based on the container run options etc.
```

Comment:
Container images generally consist of distribution packages and some scripts that help start the container properly. For example, a container with a MariaDB database typically consists of a set of RPMs, such as mariadb-server, that provides the main functionality and some scripts that handle initialization, setup, etc.